### DISCUSSION TITLE
BUILD IPA-DEBUG VS IPA-AD-HOC (BLACK SCREEN)

### DISCUSSION THREAD

**Discussion by AndersonAge** [airsdk/Adobe-Runtime-Support#3388] [Q&A]
hello, I'm having a problem with iOS.
When I do an adt ipa-debug everything works correctly, but when I try ipa-ad-hoc or ipa-app-store the application gets a black screen and nothing happens.
I don't know what to do, the debug mode works fine, but the production mode doesn't.

Has anyone had this problem?

For android it's okay, debug or production works correctly. 
IOS Debug mode too, problem is when build ipa-ad-hoc or ipa-app-store.

I've tried everything and nothing works, but when I go back to ipa-debug, everything works fine.

Using AIR SDK 51.x

**Comment by ajwfrost** [comment]
I've not come across this sort of behaviour. Are you able to connect the phone to the mac and run the Console app to see if there are any warnings/errors displayed? Or if you try connecting it to Scout, does that show whether anything is actually running?

thanks
